test/map.test.js

```diff
@@ -29,3 +29,23 @@ tape('gamma transfer function', t => {
 	t.equal(formatHex(brighten('#cc0033')), '#ff0070');
 	t.end();
 });
+
+tape(
```

Comment:
@danburzo hi!

Before this commit: https://github.com/Evercoder/culori/pull/166/commits/7c0ff5a05c094a40962282e15907b70c553a6557 the following code throws an error:

```javascript
mapper(mapTransferGamma(2.2), null)(
  'color(--okhsv 29.2 0.9 0.9)'
)
```

What do you think about [this bugfix](https://github.com/Evercoder/culori/pull/166/commits/7c0ff5a05c094a40962282e15907b70c553a6557#diff-49388a0ca466f7c0f2b6502d32934815a6fad08beb14688f86e75aabd035ca9dR10)?